decavision/model_testing/testing.py

```diff
@@ -253,3 +256,198 @@ def generate_classification_report(self, path):
         print('Labels loaded')
         # Show classification report
         print(classification_report(cls_true, cls_pred, target_names=labels, digits=4))
+
+
+
+class ModelTesterMultilabel:
+    """
+    Class to use a trained multilabel image classification model on some images. Using this
+    when working with a TPU disables eager execution.
+
+    Arguments:
+        model (str): path to trained model
+        json_file (str): path to json file containing image ids and their associated labels
```

Comment:
I would actually use the json as an argument in the methods that need it directly. This way people could use the same class and call a method like `classify_images` for different datasets.